LDK/resources/web/LDK/Utils.js

```diff
@@ -556,7 +560,9 @@ LDK.Utils = new function(){
             }
 
             if (subjectArray.length > 0) {
-                subjectArray = Ext4.unique(subjectArray);
+                if (removeDuplicates) {
```

Comment:
admittedly picky, but cant you put if if(!!removeDuplicates)? and skip the removeDuplicates==null check above?

Reply:
For the case in question, that would be !!null, which evaluates to false. We want null to be transformed to true.

Reply:
yes, clearly a typo on my part.  anyway, I think the reason this struck me was party that it's using javascript's sloppy comparisons.  under the condition it's really testing (no argument passed), isnt 'undefined' the value we expect to get?  we have this awkward/imprecise comparison (null is not really undefined).

I assume there are probably some callers elsewhere, but wouldnt making the argument 'preserveDuplicates' be more elegant?  This way the no-arg / default case would be false, which is our current behavior.  We just need to test "if(!preserveDuplicates)"

Reply:
I flipped the condition